### DISCUSSION TITLE
#14359 Does it improve performance?

### DISCUSSION THREAD

**Discussion by yuanfangcc** [openzfs/zfs#15238] [Q&A]
Does #14359 's modifications affect the read/write performance of arc ? Has anyone tested it yet?  Improved or degraded performance? This will be the basis for me to upgrade or not.Does this menu affect the read/write performance of arc modifications?

**Comment by amotin** [comment]
"read/write performance of arc" is a very vague definition.  The mentioned PR only changes what ARC caches, not how specific read/write operations are handling.  But there were other improvements for different aspects of ZFS performance, so please, respect the work of developers provided to you for free and upgrade.

**Reply by yuanfangcc** [reply]
Well, thank you for your answer.
